ci/Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM python:3.6-alpine
 
 ENV PYTHONDONTWRITEBYTECODE 1
 
-RUN apk add --no-cache build-base git gcc make libffi-dev openssl-dev && rm -rf /var/cache/apk/*
+RUN apk add --no-cache build-base git gcc make libffi-dev openssl-dev libmagic && rm -rf /var/cache/apk/*
```

Comment:
I think it will work, but my theory is that simply rebuilding the image would work as newer major release of python:3.6-alpine is now based the [3.14 dependencies](https://github.com/alpinelinux/docker-alpine/commit/6cde5b1c2838b647382fdbc71b5d3cef54c5c157) and that will include the proper version of libmagic (whereas 3.13 did not). It does not hurt though to explicitly mention `libmagic` as our deployment really depend on it and we'll kick in the rebuild that way.

![image](https://user-images.githubusercontent.com/805567/126856592-238149e6-6b58-41b6-a023-39ed92962d37.png)